solver/src/solver/paraswap_solver.rs

```diff
@@ -57,40 +60,132 @@ impl ParaswapSolver {
     }
 }
 
+fn is_retryable_err(err: &SettlementError) -> bool {
+    match err {
+        SettlementError::Paraswap(ParaswapResponseError::PriceChange) => {
+            tracing::warn!("Paraswap Price Change");
```

Comment:
nanonit: it's uncommon to print in an `is_x` method. I think the print should happen below (I will point out where)